doc/api/net.md

```diff
@@ -246,168 +352,217 @@ server.on('error', (e) => {
 });
 ```
 
-(Note: All sockets in Node.js are set `SO_REUSEADDR`.)
+*Note*: All `net.Socket` objects in Node.js are set to use the `SO_REUSEADDR`
+flag by default.
 
 ### server.listening
 <!-- YAML
 added: v5.7.0
 -->
 
-A Boolean indicating whether or not the server is listening for
-connections.
+A boolean indicating whether or not the server is listening for connections.
 
 ### server.maxConnections
 <!-- YAML
 added: v0.2.0
 -->
 
-Set this property to reject connections when the server's connection count gets
-high.
+The `server.maxConnections` property can be set to specify the maximum number
+of connections the server will accept. Once the current number of connections
+reaches this threshold, all additional connection requests will be rejected.
+The default is `undefined`.
```

Comment:
not sure if it can be deleted or re-assigned, or what happens when set to a value lower than current number of connections, or if it has to be set before .listen() is called on the server.